WhoopTriggerzDesign/Utilities/Colors+Ext.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Colors+Ext.swift
+//  WhoopTriggerzDesign
+//
+//  Created by Jessica Perez on 5/10/23.
+//
+
+import SwiftUI
+
+extension Color {
+    static let sliderColor = Color("blackColor")
```

Comment:
I think we might need to adjust this value to better match the slider black color

Reply:
Resolved in 1d0cfdf